src/hal/motor.c

```diff
@@ -34,25 +53,53 @@ void power_left(int32_t power)
  */
 void power_right(int32_t power)
```

Comment:
This function is practically the same as power_left. Should we put the common code in another function?

Reply:
We can start a discussion. The common function should receive what driver channels to control and should update the right "saturated" variable.

IMHO: I would leave the code as is